src/lib/meminfo/meminfo.c

```diff
@@ -36,6 +33,10 @@
 #endif
 #include <string.h>
 
+#if HAVE_SYS_SYSCTL_H && !defined(_WIN32) && !defined(__linux__)
```

Comment:
We should check "defined(HAVE_SYS_SYSCTL_H)", not "SYS_SYSCTL_H": appveyor is failing with this error:

```
bash.exe : ../src/lib/meminfo/meminfo.c:36:5: error: "HAVE_SYS_SYSCTL_H" is not defined, evaluates to 0 [-Werror=undef]
```